src/shared/Microsoft.Git.CredentialManager/Git.cs

```diff
@@ -12,6 +15,20 @@ public interface IGit
         /// <param name="level">Configuration level filter.</param>
         /// <returns>Git configuration.</returns>
         IGitConfiguration GetConfiguration(GitConfigurationLevel level);
+
+        /// <summary>
+        /// Create a Git process that's ready to run.
+        /// </summary>
+        /// <returns>The Git process.</returns>
+        Process CreateProcess(string args);
```

Comment:
We can drop this from the interface now that we've got `InvokeHelperAsync` on the interface.. the `GitProcess` implementation can still call `CreateProcess`, and we don't need to worry about the `TestGit` implementation at all.

Reply:
Oh, sure thing. I added it for parallelism with `InvokeHelperAsync` but yeah, YAGNI.